taxi/forms.py

```diff
@@ -0,0 +1,49 @@
+from django import forms
+from django.contrib.auth import get_user_model
+from django.contrib.auth.forms import UserCreationForm
+
+from taxi.models import Driver, Car
+
+
+class LicenseValidator:
+    def clean_license_number(self):
```

Comment:
The method `clean_license_number` should be renamed to `validate_license_number` to better reflect its purpose and avoid confusion with Django's `clean` methods.